config/cgimap.xml.orig

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<cgimap copyright="OpenStreetMap and contributors" attribution="http://www.openstreetmap.org/copyright" license="http://opendatacommons.org/licenses/odbl/1-0/">
+  <settings>
+    <map_nodes maximum="50000"/>
+    <area maximum="0.25"/>
+    <changeset maximum_elements="10000"/>
+    <waynodes maximum="2000"/>
+    <relation_members maximum="32000"/>
+    <element_tags maximum="5000"/>
```

Comment:
These two options were added so that the current PR "Enforce limits for tags and relation members #174" is already integrated.
https://github.com/zerebubuth/openstreetmap-cgimap/pull/174